test/ci/create_aws_vm.yml

```diff
@@ -0,0 +1,50 @@
+- name: Create a VM
+  ec2:
+    region: "{{ aws_region }}"
+    key_name: "{{ aws_key }}"
+    instance_type: "{{ outer_item.aws_flavor }}"
+    image: "{{ outer_item.aws_image | default(aws_image) }}"
+    wait: yes
+    group: "{{ outer_item.aws_security_group }}"
+    count: 1
+    vpc_subnet_id: "{{ aws_subnet }}"
+    assign_public_ip: yes
+    instance_tags:
+      Name: "{{ outer_item.name_template }}-{{ item }}"
+      "kubernetes.io/cluster/{{ aws_cluster_id }}": "true"
```

Comment:
TODO: This needs fixing, as this var doesn't get templated for some reason

Reply:
@vrutkovs i suspect the reason for that is because is not a json. What i ended up doing internally for same thing was:

```
- name: Set ec2_tags fact for EC2 instances
   set_fact:
     ec2_tags: |
       {
          "kubernetes.io/cluster/{{ openshift_clusterid }}": "{{ openshift_clusterid }}",
          "Name": "{{ name }}",
```
followed by
```
- name: Add tags to {{ name }} instance
   ec2_tag:
     region:  "{{ aws_region }}"
     resource: "{{ instance_id }}"
     state: present
     tags: "{{ ec2_tags }}"
```

hth

Reply:
also i believe you want to change the tag's value to one of the 2 values https://github.com/kubernetes/kubernetes/blob/master/pkg/cloudprovider/providers/aws/tags.go#L43-L51

Reply:
Yeah, that did the trick, thanks!

Not sure if we care about `owned` vs. `shared`, might add this later